rootfs/chainid.go

```diff
@@ -0,0 +1,50 @@
+package rootfs
+
+import digest "github.com/opencontainers/go-digest"
+
+// NOTE(stevvooe): These are copy-pasted from the opencontainers/image-spec
+// project. Once https://github.com/opencontainers/image-spec/pull/486 is
+// merged, we can use those from the upstream.
+
+// ChainID takes a slice of digests and returns the ChainID corresponding to
```

Comment:
@samuelkarp seems defined in https://github.com/opencontainers/image-spec/blob/master/config.md#layer-chainid